tests/integrationtest/r/globalindex/mem_index_lookup.result

```diff
@@ -55,6 +56,39 @@ select * from t partition(p0, p1) use index(idx1) where b <= 2;
 a	b
 1	2
 5	1
+explain select * from t use index(idx) where a > 2;
+id	estRows	task	access object	operator info
+Projection_5	3333.33	root		globalindex__mem_index_lookup.t.a, globalindex__mem_index_lookup.t.b
+└─UnionScan_6	3333.33	root		gt(globalindex__mem_index_lookup.t.a, 2)
+  └─IndexLookUp_10	3333.33	root	partition:all	
```

Comment:
Should we have the `partition:...` clause in the `access object` for global indexes?

Reply:
```
mysql> explain select b from t use index(idx) where a < 10;
+---------------------------------+---------+-----------+-----------------------+-------------------------------------------------+
| id                              | estRows | task      | access object         | operator info                                   |
+---------------------------------+---------+-----------+-----------------------+-------------------------------------------------+
| Projection_4                    | 3323.33 | root      |                       | test.t.b                                        |
| └─IndexLookUp_10                | 3323.33 | root      | partition:p0          |                                                 |
|   ├─Selection_9(Build)          | 3323.33 | cop[tikv] |                       | in(_tidb_tid, 514)                              |
|   │ └─IndexRangeScan_6          | 3323.33 | cop[tikv] | table:t, index:idx(a) | range:[-inf,10), keep order:false, stats:pseudo |
|   └─TableRowIDScan_7(Probe)     | 3323.33 | cop[tikv] | table:t               | keep order:false, stats:pseudo                  |
+---------------------------------+---------+-----------+-----------------------+-------------------------------------------------+
5 rows in set (0.00 sec)
```

We could get prune information from `access object` and `Selection`. And I think `access object` is more intuitive.